apps/site/pages/docs/_meta.json

```diff
@@ -17,7 +17,13 @@
     }
   },
   "customizing": {
-    "title": "Customizing",
+    "title": "Customizing Components",
```

Comment:
I think we can create a sub page under Customizing with `Customizing Pages`. WDYT?